src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -1898,7 +1928,295 @@ subroutine get_conus_netcdf(date_time, met_path, N_catd, tile_coord, &
+    ! space)
+    do kk=1,N_era5_compressed
+       
+       land_i_era5(kk)=mod((p2g(kk)),era5_grid_N_lon)
+       if(land_i_era5(kk) .eq. 0) land_i_era5(kk)=era5_grid_N_lon
+       
+       land_j_era5(kk)=floor((p2g(kk))/float(era5_grid_N_lon))+1
+       if (mod(p2g(kk),era5_grid_N_lon) .eq. 0) land_j_era5(kk)=land_j_era5(kk)-1
+       
+    end do
+    
+    ! close NC file
+    ierr = NF90_CLOSE(ncid)
+
+    ! Get forcing data geolocation (to be used for Exclude list creation)
+    ! Read in LON and LAT values from only one Forcing file. This information
+    ! is static! Just make sure to use a file where the info is available (since not
+    ! all ERA5 LIS files carry this info)
+
+    fname = trim(met_path) // '/FORCING_201807.nc'
+
+    if(root_logit) write (logunit,*) 'get ERA5 LIS file forcing data lat/lons from ' // trim(fname)
+
+    ierr = NF90_OPEN(fname,NF90_NOWRITE,ncid)
+    if (ierr/=0) then
+       err_msg = 'error opening netcdf file'
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, err_msg)
+    end if
+
+    ! read Longitude and Latitude from nc file 
+    ierr = NF90_INQ_VARID(ncid,'LON',era5_varid)
+    ierr = NF90_GET_VAR(ncid, era5_varid, era5_lon)
+    ierr = NF90_INQ_VARID(ncid,'LAT',era5_varid)
+    ierr = NF90_GET_VAR(ncid, era5_varid, era5_lat)
+
+    ! populate lat/lon 2D arrays
+    do n=1,N_era5_compressed
+       era5_lat2D(land_i_era5(n), land_j_era5(n) ) = era5_lat(n)
+       era5_lon2D(land_i_era5(n), land_j_era5(n) ) = era5_lon(n)
+    end do
+
+    ! close NC file
+    ierr = NF90_CLOSE(ncid)
+
+    ! read Forcing data (for the date corresponding to LIS file) 
+
+    ! open file
+    fname = trim(met_path) // '/FORCING_' // YYYY // MM // '.nc'
+
+    if(root_logit) write (logunit,*) 'opening ' // trim(fname)
+
+    ierr = NF90_OPEN(fname,NF90_NOWRITE,ncid)
+
+    if (ierr/=0) then
+       err_msg = 'error opening netcdf file'
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, err_msg)
+    end if
+
+    ! loop through forcing variables
+
+    do era5_var = 1,N_era5_vars
+       
+       ierr = NF90_INQ_VARID(ncid,trim(era5_name(era5_var)),era5_varid)   ! get var ID
+       ierr = NF90_GET_VAR(ncid,era5_varid,tmp_vec,start,icount)          ! read variable
+       
+       ! put data on global grid
+       
+       tmp_grid  = nodata_forcing    ! initialize 
+
+       do n=1,N_era5_compressed
+          tmp_grid(land_i_era5(n),land_j_era5(n)) = tmp_vec(n)
+       end do
+
+       ! interpolate to tile space
+       do k=1,N_catd
+          ! if the GEOS tile is not within the 0.125*sqrt(2) deg of the matching ERA5
+          ! tile, asign a missing value to it (this will trigger Exclude list flagging)
+          ! otherwise, feed with colocated tmp_grid element
+
+          geos2era_tile_SQdistance =                                             &
+             (tile_coord(k)%com_lon+0.0001 - era5_lon2D(i_ind(k),j_ind(k)))**2 + &
+             (tile_coord(k)%com_lat+0.0001 - era5_lat2D(i_ind(k),j_ind(k)))**2 
```

Comment:
This ( +0.0001)  is quite awkward  to me.

Reply:
I agree, but it was an easy and quick way to fix the issue. I am open to suggestions.

Reply:
This block is almost certainly going away, so should not be an issue. 
Further up, the +0.0001 is going to be needed unless someone has a better solution.
But I still think we should at an "if (EASE grid tile space)" condition if that's possible.

Reply:
This entire section will be removed, since the distance match is not needed anymore.